osu.Game/Rulesets/Mods/IApplicableToBeatmap.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Beatmaps;
+
+namespace osu.Game.Rulesets.Mods
+{
+    /// <summary>
+    /// Interface for a <see cref="Mod"/> that applies changes to a <see cref="Beatmap"/>.
```

Comment:
Similar to the other comment, this should mention when this application happens. Something like `This <see cref="Mod"/> is applied after the conversion and post-processing of the <see cref="Beatmap"/> has completed.`